### ISSUE TITLE
clj-kondo crashes when there is a :require :rename and ":clj-kondo/ignore" is used

### ISSUE THREAD

**Issue by lucywang000** [clj-kondo/clj-kondo#1210]
Minimal repro:

```
$ echo '(ns foo (:require [bar :rename {v1 v2}])) #_{:clj-kondo/ignore [:unresolved-symbol]} (+ 1 2)'  | clj-kondo --lint -
Unexpected error. Please report an issue.
java.lang.NullPointerException
        at clojure.lang.Numbers.ops(Numbers.java:1068)
        at clojure.lang.Numbers.gt(Numbers.java:259)
        at clj_kondo.impl.findings$ignored_QMARK_.invokeStatic(findings.clj:23)
        at clj_kondo.impl.findings$reg_finding_BANG_.invokeStatic(findings.clj:49)
        at clj_kondo.impl.linters$lint_unused_namespaces_BANG_.invokeStatic(linters.clj:454)
        at clj_kondo.core$run_BANG_.invokeStatic(core.clj:151)
        at clj_kondo.main$main.invokeStatic(main.clj:117)
        at clj_kondo.main$main.doInvoke(main.clj:113)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clj_kondo.main$_main$fn__15849.invoke(main.clj:136)
        at clj_kondo.main$_main.invokeStatic(main.clj:136)
        at clj_kondo.main$_main.doInvoke(main.clj:134)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clj_kondo.main.main(Unknown Source)

$ clj-kondo --version
clj-kondo v2021.03.03
```

This also affects clojure-lsp.

**Comment by borkdude** [comment]
@lucywang000 Thanks. This could be a duplicate of https://github.com/clj-kondo/clj-kondo/issues/1195. Can you test with master?

**Comment by lucywang000** [comment]
@borkdude Thanks for the quick reply! I did test with the master. 

```
$ git log -1
commit 7bad82031e3b466ece4562baccece587f1d9345e
Author: Andrey Fedorov <oran9e.red@gmail.com>
Date:   Wed Mar 10 17:35:51 2021

    Add tests and update hooks.md (#1209)

$ lein clj-kondo --lint  /tmp/foo.cljs
Unexpected error. Please report an issue.
java.lang.NullPointerException: Cannot invoke "Object.getClass()" because "x" is null
        at clojure.lang.Numbers.ops(Numbers.java:1018)
        at clojure.lang.Numbers.gt(Numbers.java:234)
        at clj_kondo.impl.findings$ignored_QMARK_.invokeStatic(findings.clj:23)
        at clj_kondo.impl.findings$ignored_QMARK_.invoke(findings.clj:11)
        at clj_kondo.impl.findings$reg_finding_BANG_.invokeStatic(findings.clj:51)
        at clj_kondo.impl.findings$reg_finding_BANG_.invoke(findings.clj:44)
        at clj_kondo.impl.linters$lint_unused_namespaces_BANG_.invokeStatic(linters.clj:452)
        at clj_kondo.impl.linters$lint_unused_namespaces_BANG_.invoke(linters.clj:419)
        at clj_kondo.core$run_BANG_.invokeStatic(core.clj:153)
        at clj_kondo.core$run_BANG_.invoke(core.clj:51)
        at clj_kondo.main$main.invokeStatic(main.clj:125)
        at clj_kondo.main$main.doInvoke(main.clj:113)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:657)
        at clojure.core$apply.invoke(core.clj:652)
        at clj_kondo.main$_main$fn__25060.invoke(main.clj:136)
        at clj_kondo.main$_main.invokeStatic(main.clj:136)
        at clj_kondo.main$_main.doInvoke(main.clj:134)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at clojure.lang.Var.invoke(Var.java:385)
        at user$eval8437.invokeStatic(form-init690982162513828124.clj:1)
        at user$eval8437.invoke(form-init690982162513828124.clj:1)
        at clojure.lang.Compiler.eval(Compiler.java:7062)
        at clojure.lang.Compiler.eval(Compiler.java:7052)
        at clojure.lang.Compiler.load(Compiler.java:7514)
        at clojure.lang.Compiler.loadFile(Compiler.java:7452)
        at clojure.main$load_script.invokeStatic(main.clj:278)
        at clojure.main$init_opt.invokeStatic(main.clj:280)
        at clojure.main$init_opt.invoke(main.clj:280)
        at clojure.main$initialize.invokeStatic(main.clj:311)
        at clojure.main$null_opt.invokeStatic(main.clj:345)
        at clojure.main$null_opt.invoke(main.clj:342)
        at clojure.main$main.invokeStatic(main.clj:424)
        at clojure.main$main.doInvoke(main.clj:387)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.lang.Var.applyTo(Var.java:702)
        at clojure.main.main(main.java:37)

$ cat /tmp/foo.cljs
(ns foo
  (:require [bar :rename {v1 v2}]))

#_:clj-kondo/ignore
(+ 1 2)
```

**Comment by borkdude** [comment]
Repro:

```
$ clojure -M:clj-kondo --lint - <<< '(ns foo (:require [bar :rename {v1 v2}])) ::bar/x'
<stdin>::: warning: #'bar/v1 is referred but never used
``` 

Cause: finding without row/col.

**Comment by borkdude** [comment]
Fixed on master.
